configure.ac

```diff
@@ -72,6 +72,18 @@ AS_IF([test "x$enable_turn_off_ucd90160_access" != "xyes"],
       AC_DEFINE_UNQUOTED([UCD90160_DEVICE_ACCESS], ["$UCD90160_DEVICE_ACCESS"], [Turn off UCD90160 hardware access])
 )
 
+AC_ARG_ENABLE([turn-off-mihawkcpld-access],
```

Comment:
when you upstream this code, you can just take out this part.  AT the time we thought we needed it for the UCD, but now I'm not really sure of the reasoning.

Reply:
@spinler ok, I will adjust this part again when I upstream this code.